lib/container/container.go

```diff
@@ -0,0 +1,93 @@
+// package container contains convenience functions for manipulating containers
+package container
+
+import (
+	"io"
+	"log"
+
+	"github.com/fsouza/go-dockerclient"
+)
+
+func RawStartContainer(opts docker.CreateContainerOptions) (string, error) {
+	client, err := docker.NewClient("unix:///var/run/docker.sock")
```

Comment:
This is a hackish version I wrote up a while ago https://github.com/peter-edge/go-dockerutils/blob/master/dockerutils.go#L55 but I wouldn't recommend directly using it, I may refactor it or delete it soon.